pkg/clients/imagebuilder/client.go

```
@@ -390,7 +392,9 @@ func (c *Client) GetMetadata(image *models.Image) (*models.Image, error) {
 	}
 
 	var metadata Metadata
-	json.Unmarshal(respBody, &metadata)
+	if err := json.Unmarshal(respBody, &metadata); err != nil {
+		c.log.Error("Error while trying to unmarshal ", metadata)
```

Can we return this error?